aidies-project/client/src/components/Navigation.js

```diff
@@ -125,6 +136,18 @@ function Navigation(props) {
                     : <LogoutButton/>} </div>
                 </div>
             </div>
+            <div className="category-filter">
+                {categories.map(category => <div key={category.id} className="category-button"><ToggleButton
+                                            id={category.name}
+                                            value="check"
+                                            selected={selected}
+                                            onChange={handleToggle}
+                                            >
+                                            {category.name}
+                                            </ToggleButton></div>
+                                )}
+                
+            </div>
```

Comment:
Minor spacing suggestion
```suggestion
            <div className="category-filter">
                {categories.map(category => (
                    <div key={category.id} className="category-button">
                      <ToggleButton
                          id={category.name}
                          value="check"
                          selected={selected}
                          onChange={handleToggle}
                      >
                          {category.name}
                      </ToggleButton>
                    </div>
                ))}
            </div>
```